### PR TITLE
[DC-758] Delete run-catalog-workflow and move to using existing workspaces

### PR DESCRIPTION
**Pull request by s-rubenstein**
### What
- Deletes the `run-catalog-workflow` test and moves `export-tdr-dataset-to-new-workspace` and `export-workspace-dataset-to-new-workspace` to use existing workspaces

### Why
- We were duplicating a lot of our tests - we probably only need one of `run-catalog-workflow` and `export-tdr-dataset-to-new-workspace`
- We decided that creating a new workspace was not an important part of this test

### Testing strategy
- [ ]

### CODE REVIEW

**Review by pshapiro4broad — APPROVED** [review]
Looks OK, I wonder if coverage will change much

**Comment by s-rubenstein** [comment]
I don't think coverage numbers will change - those are strictly unit coverage, IIRC

**Review by nawatts — APPROVED** [review]

File: integration-tests/utils/catalog-utils.js [hunk 1]
```diff
@@ -18,47 +18,14 @@ const linkDataToWorkspace = async (page, testUrl, token, datasetName) => {
   await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Prepare for analysis' })) });
 };
 
-const testExportToNewWorkspace = async (billingProject, page, testUrl, token, datasetName) => {
+const testExportToWorkspace = async (billingProject, page, testUrl, token, datasetName, workspaceName) => {
```
Comment (nawatts) [inline integration-tests/utils/catalog-utils.js:21]: It's really nice how much simpler this test became.

**Comment by nawatts** [comment]
> I don't think coverage numbers will change - those are strictly unit coverage, IIRC

That is correct.

**Pull request merged**